Description: Fixes blinking pointer and 100% CPU usage with GTK applications
Author: Tao Nelson <taobert@gmail.com>
Debian-Bug: 266118

Index: unclutter/unclutter.c
===================================================================
--- unclutter.orig/unclutter.c	2013-04-30 01:32:16.931767241 +0200
+++ unclutter/unclutter.c	2013-04-30 01:32:16.915766992 +0200
@@ -399,7 +399,9 @@
 		do{
 		    XNextEvent(display,&event);
 		}while(event.type!=LeaveNotify &&
-		       event.type!=FocusOut &&
+		    /* Some gtk applications seem not to like this:
+		     * event.type!=FocusOut && 
+		     */
 		       event.type!=UnmapNotify &&
 		       event.type!=ConfigureNotify &&
 		       event.type!=CirculateNotify &&
